### ISSUE TITLE
Running in a container

### ISSUE THREAD

**Issue by Tibormach** [mamba-org/provision-with-micromamba#46]
I am trying use your action in a container, specifically [this](https://github.com/iterative/cml/pkgs/container/cml) one by Iterative in order to create a workflow with CML.

Unfortunately, I am running into a problem with `.bashrc` (or rather with the user name if I understand the issue correctly)

The issue is replicated in [this repository](https://github.com/Tibormach/provision_with_micromamba_test).
There is a workflow which should set up an environment with micromamba on a pull request.
There are two branches (apart from main), the workflow works as intended in [the one where the CML container is not used](https://github.com/Tibormach/provision_with_micromamba_test/pull/1) and it fails on the following error in [the one where the container is used](https://github.com/Tibormach/provision_with_micromamba_test/pull/2).

![image](https://user-images.githubusercontent.com/56956998/159132710-52b9a75c-4311-4b73-a951-6c3a2bc41c77.png)

It is possible this is not an issue with provision-with-micromamba and that I am doing something wrong (i.e. I don't understand how to set up up correctly when using the container).

Thanks for the help!

**Comment by wolfv** [comment]
Not sure exactly how you are using this. But there is also a micromamba container here that might be better for your use case? 

https://github.com/mamba-org/micromamba-docker/

**Comment by Tibormach** [comment]
@wolfv I need to run the job in the specific container used in [this workflow](https://github.com/Tibormach/provision_with_micromamba_test/blob/not_working/.github/workflows/pr.yml)

That is a CML container and the point of the job (not the one in that example repository but when I use it in practice) is to re-run an ML pipeline and log all the relevant metrics (which is more or less what CML is for).

In the job I need to set up a virtual environment. I can do it "manually" with a bash script, but the environment is relatively big and takes some 4 minutes to set up. I want to reduce the time to some 20 seconds which I can with provision-with-micromamba and caching when not run in the container.

So basically, my main usecase is caching the virtual environment but now that I think about it again I guess I'd need to cache the container as well (though maybe not, I am not sure how exactly the caching is done), or create an image which includes the virtual environment already, so I am not quite sure I can actually use provision-with-micromamba here...
